package.json

```diff
@@ -306,6 +309,8 @@
     "test-visual-run": "percy exec -- yarn test-cypress-run --spec \"./frontend/test/metabase-visual/**/*.cy.spec.js\"",
     "test-visual-open": "percy exec -- yarn test-cypress-open",
     "test-visual": "yarn build && ./bin/build-for-test && yarn test-visual-run",
+    "clean:cypress:reports": "rm -R -f cypress/",
```

Comment:
I have a few issues with this:

1. Are you sure that deleting the whole Cypress folder is a good idea?
2. Even if it is, the name of the command and the actual command don't match. The intention matters.
    - Either rename the command to convey that we are deleting the whole folder or
    - delete only cypress/reports folder
    
But the biggest problem is that you defined it here, but never called/used it **anywhere** else in the code. That means that if someone forgets to manually clean up before they generate a report, they are going to create a mess.

Tried it locally by doing this:
- ran spec A => generated the HTML report (includes JSON from the spec A)
- ran spec B => generated the HTML report (includes the result from both JSONs from **spec A and spec B**)

Reply:
To be able to have the behavior you expect we would have to:

1-) Delete the `reportFilename`.json everytime we do a `mochawesome-merge` or/and change the `overwrite` to `true` on `cypress.json`. That way, we will always have new `json` on new runs AFTER generating the report. And also, the updated version of the spec. But if someone runs and doesn't generate the HTML report and runs another spec and then generates it, the report will have more than one spec on it. 

2-) Add the deletion of the `jsons` on the runtime of a spec.

If you are seeing anything different, please, let me know.